itest/common.go

```diff
@@ -30,3 +33,51 @@ func assertBlindedPathEqual(t *testing.T, expected,
 			"encrypted data", i)
 	}
 }
+
+// consumeOnionMessage sets up a closure that can be used to consume messages
+// delivered from an onion message subscription client.
+func consumeOnionMessage(wg *sync.WaitGroup,
```

Comment:
Make note why we run this in a goroutine (test would hang if we failed to receive otherwise)